front/src/common/Map/Search/MapSearchSignal.tsx

```diff
@@ -114,19 +113,19 @@ const MapSearchSignal = ({ updateExtViewport, closeMapSearchPopUp }: MapSearchSi
 
   const updateSearch = async (infraIDPayload: number) => {
     const settings = selectedSettings.map((setting) => REVERSED_SIGNAL_SETTINGS_DISPLAY[setting]);
-    const payload: searchPayloadType = getPayload(
+    const payload: SearchPayload = getPayload(
```

Comment:
No need for this type :) Typescript knows what it is
```suggestion
    const payload = getPayload(
```